ui/public/locales/en.json

```
@@ -1600,8 +1600,8 @@
 "label.userdatadetails": "Userdata details",
 "label.userdataparams": "Userdata parameters",
 "label.userdatapolicy": "Userdata link policy",
-"label.userdata.text": "Userdata Text",
-"label.userdata.registered": "Userdata Registered",
+"label.userdata.text": "Manual Userdata",
```

```suggestion
"label.userdata.text": "Manual Userdata entry",
```
no -1, just a polish suggestion.